nmrfx-plugin-api/src/main/java/org/nmrfx/plugin/api/EntryPoint.java

```diff
@@ -22,5 +22,5 @@
  * The obvious ones would be menus and scenes.
  */
 public enum EntryPoint {
-    STARTUP, MENU_PLUGINS, STATUS_BAR_TOOLS
+    STARTUP, MENU_PLUGINS, STATUS_BAR_TOOLS, MENU_FILE
```

Comment:
small detail but I'd prefer if we kept the similar entrypoints closer together.

something like that:
```suggestion
    STARTUP, MENU_FILE, MENU_PLUGINS, STATUS_BAR_TOOLS
```

Reply:
made this change